### PR TITLE
Comply to new Actions primary/title standard

### PR DESCRIPTION
**Pull request by skjnldsv**
Needs https://github.com/nextcloud/nextcloud-vue/pull/1062

image

### CODE REVIEW

File: src/views/Create.vue [hunk 1]
```diff
@@ -71,9 +71,10 @@
 			<!-- Add new questions toolbar -->
 			<div class="question-toolbar" role="toolbar">
 				<Actions ref="questionMenu"
-					v-tooltip="t('forms', 'Add a question to this form')"
 					:aria-label="t('forms', 'Add a question to this form')"
```
Comment (jancborchardt) [inline src/views/Create.vue:74]: Now that we have a proper button with text, we can remove the additional aria-label, no?

Comment (skjnldsv) [reply src/views/Create.vue:74]: They're different

**Comment by skjnldsv** [comment]
> * Text in our buttons is always bold, especially in the primary actions

Should be for all right?
https://github.com/nextcloud/nextcloud-vue/pull/1062

File: src/views/Create.vue (hunk 1 above)
Comment (skjnldsv) [reply src/views/Create.vue:74]: Ah, you meant accessibility speaking!
Yes we don't need it anymore, I was thinking about the technical, they're different props :stuck_out_tongue_closed_eyes:

**Comment by skjnldsv** [comment]
Done in https://github.com/nextcloud/nextcloud-vue/pull/1062 for the bold part, please review there Jan

**Comment by jancborchardt** [comment]
The Vue fix was merged, can we merge this then or do we need a release of the components?

**Comment by skjnldsv** [comment]
> we need a release of the components?

this

**Comment by skjnldsv** [comment]
Pushed @nextcloud/vue 2.0.0 update

**Pull request merged**